src/pages/blog/2024-07-22-graphql-local-initiative.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "Launching the GraphQL Local Initiative"
+tags: ["blog"]
+date: 2024-07-22
+byline: Benjie Gillam and Jory Burson
+---
+
+The GraphQL Foundation's primary mission is to ensure the continued evolution of
+the GraphQL specification and reference implementations. As part of this
+mission, we strive to provide collaboration infrastructure that fosters a
+culture of community and innovation. Local get-togethers play a crucial role in
+this, bringing together developers, library maintainers, vendors, and
+enthusiasts to share knowledge and collaborate.
+
+This is why we're pleased to announce the launch of the **GraphQL Local
```

Comment:
I wonder if we want to say something like, "That's why we've expanded our support for local communities with the launch of the GraphQL Local Initiative" - so we're acknowledging the past efforts of the meetup pro network but also saying, this replaces that program